projects/Mallard/scripts/get-settings-pages-html.js

```diff
@@ -10,7 +10,6 @@ const capiApiKey = process.env.CAPI_API_KEY
  */
 const settingsPagePathsToFetch = {
     'help/2019/sep/12/the-guardian-daily-beta-faqs': 'faq',
-    'help/privacy-policy': 'privacy-policy',
```

Comment:
We not scraping the site anymore? Should we be consistent with these edge pages?

Reply:
Nope - we have a specific privacy policy for this app that isn't available online. (If it is in the future we can point it at this again)

Reply:
Blish hand-cranked it from the doc.

Reply:
Moved the html into a variable in the component to make clear.